### DISCUSSION TITLE
Netplay with Pokemon (GBC) - Onion OS 4.2.3

### DISCUSSION THREAD

**Discussion by OldSchoolGamr** [OnionUI/Onion#1378] [Support]
I'm on Onion OS 4.2.3 and using Netplay with Pokemon Crystal & Gold. I was able to successfully trade pokemon but for some reason after trading and saving, loading the game defaults back to the pre-trade state/save-point when loading the game using TGB Dual, however if i switch the core back to Gambatte it loads the post-save state/save-point. 

Does anyone know of a walkthrough or able to direct me to some type of step-by-step guide that shows me how to properly use Netplay to trade pokemon and then save on both miyoo mini plus devices so that i'm not having to switch between Gambatte & TGB Dual? 

Just got Miyoo Mini+ and new to this... sorry for the noob question. (also been about 15 years since I've played around with Linux files and devices, so I'm also very rusty).

**Comment by schmurtzm** [comment]
Hi, yes it requires some explanations ;)

First you have to know that the core selection is now automatic for Netplay : the Netplay default core is now selected automatically thank to [netplay_cores.conf](https://github.com/OnionUI/Onion/blob/main/static/build/.tmp_update/config/netplay_cores.conf).

But when you don't use Netplay, the default GB emulator is Gambatte.... So we made something to avoid to the community to switch cores all the time, all is automatic: when you play normally to Pokemon it will start with Gambatte but when you'll run Easy Netplay the saves from Gambatte will be imported in TGB Dual and after the trade the modified save will be restored in Gambatte. (So yes the most of the youtube tutorials are deprecated).

In other words: 

- backup and delete your TGB Dual save
- restore Gambatte as default core for your pokemon game (don't switch the cores anymore)
- run all your Netplay sessions from GLO menu

and it should be OK.
